### DISCUSSION TITLE
Implementing Jump Start Reinforcement Learning in DI-engine

### DISCUSSION THREAD

**Discussion by mahuangxu** [opendilab/DI-engine#340] [General]
Hope DI-engine can support JSRL (Jump Start Reinforcement Learning), an algorithm that employs two policies to solve tasks: a guide-policy, and an exploration-policy.

https://arxiv.org/abs/2204.02372 (https://jumpstart-rl.github.io/)

**Comment by PaParaZz1** [comment]
Nice proposal! We have also noticed this paper recently, and we will assign a developer for this after internal discussion. Are you willing to work together?
